src/main/java/org/gaul/s3proxy/S3ProxyHandler.java

```diff
@@ -18,6 +18,12 @@
 
 import static java.util.Objects.requireNonNull;
 
+// CHECKSTYLE:OFF
```

Comment:
I understand why these static imports are temporarily needed but please add a TODO to remove them when all methods are reparented.